frontend/src/console/components/ThemeNavigation/ThemeNavigation.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import './styles.scss';
+import {
+  ConsolePage,
+  ConsolePageCustomizationContainer,
+  NavigationMenu
+} from 'console/components';
+import { InstancesModel } from 'console/models';
+import { connect } from 'react-redux';
+import { RootState } from 'global/state';
+import { updateThemeFieldValue } from 'console/actions';
+import { Col, Row } from 'react-bootstrap';
+import { ColorInputField } from '../../../ui/components/ColorInputField';
+import messages from './displayMessages';
+import { WrappedMessage } from '../../../utils/intl';
+
+interface State {}
+
+interface ActionProps {
+  updateThemeFieldValue: Function;
+}
+
+interface StateProps extends InstancesModel {}
+
+interface Props extends StateProps, ActionProps {}
+
+export class ThemeNavigationComponent extends React.PureComponent<
+  Props,
+  State
+> {
+  private onChangeColor = (fieldName: string, newColor: string) => {
+    const instance = this.props.activeInstance;
+
+    if (instance.data) {
+      this.props.updateThemeFieldValue(instance.data.id, fieldName, newColor);
+    }
+  };
+
+  public render() {
+    const instance = this.props.activeInstance;
+    let themeData;
+
+    if (instance.data && instance.data.draftThemeConfig) {
+      themeData = instance.data.draftThemeConfig;
+    }
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        <ConsolePageCustomizationContainer>
+          <h2>
+            <WrappedMessage messages={messages} id="themeNavigation" />
+          </h2>
+
+          {themeData && themeData.version === 1 && (
+            <div className="theme-navigation-container">
+              <NavigationMenu
+                instanceData={instance.data}
+                themeData={themeData}
+              />
+              <NavigationMenu
+                instanceData={instance.data}
+                themeData={themeData}
+                loggedIn
+              />
+              <Row>
+                <p className="style-name">
+                  <WrappedMessage messages={messages} id="navigationLinks" />
+                </p>
+              </Row>
+              <Row>
+                <Col md={4}>
+                  <ColorInputField
```

Comment:
There's some issue with the `useEffect` usage in this page. When clicking on `reset`, the colors are reset to the default value, but the field contents are not updated. The same thing doesn't happen in the `Theme preview and colors` page. 
![Peek 2020-03-27 10-42](https://user-images.githubusercontent.com/27893385/77762123-cb901e80-7017-11ea-80c2-3f20d8c9a9cf.gif)
Can you check what's causing this?

Reply:
We're getting this error:
```
Warning: A component is changing a controlled input of type undefined to be uncontrolled. Input elements should not switch from controlled to uncontrolled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
```

It seems to be happening only when from/to empty string.

Reply:
That's weird, this might not happen in the other page because the backend returns default values instead of empty ones when resetting.
Can you check if there's a straightforward fix for this?